backend/views/syson-diagram-common-view/src/main/java/org/eclipse/syson/diagram/common/view/nodes/AbstractControlNodeActionNodeDescriptionProvider.java

```diff
@@ -47,6 +43,8 @@ public abstract class AbstractControlNodeActionNodeDescriptionProvider extends A
 
     private final IDescriptionNameGenerator descriptionNameGenerator;
 
+    private NodeDescription nodeDescription;
```

Comment:
Please do not store this variable. You should be able to retrieve the NodeDescription created by the `create` method with `cache.getNodeDescription(...)`.